build.sbt

```diff
@@ -556,7 +556,7 @@ lazy val commonSettings = Seq(
   },
   coursierVerbosity := 0,
   ivyLoggingLevel := UpdateLogging.Quiet // This doesn't seem to work? We see this in MiMa
-)
+) ++ xlint
```

Comment:
I think line 492 is the problem.  You're adding the right xlint here, but `-Xlint` is already in the list.